package.json

```diff
@@ -1,19 +1,47 @@
 {
   "name": "rl-js-agents",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "description": "Reinforcement learning agents in JavaScript",
-  "author": "Chris Nota <cnota@cs.umass.edu>",
```

Comment:
FYI, `npm` parses the `AUTHORS` file, which will scale-up much better than `package.json`.